src/test/java/com/thirdcc/webapp/web/rest/EventActivityResourceIT.java

```diff
@@ -110,52 +123,182 @@ public static EventActivity createEntity(EntityManager em) {
             .description(DEFAULT_DESCRIPTION);
         return eventActivity;
     }
-    /**
-     * Create an updated entity for this test.
-     *
-     * This is a static method, as tests for other entities might also need it,
-     * if they test an entity which requires the current entity.
-     */
-    public static EventActivity createUpdatedEntity(EntityManager em) {
-        EventActivity eventActivity = new EventActivity()
-            .eventId(UPDATED_EVENT_ID)
-            .startDate(UPDATED_START_DATE)
-            .durationInDay(UPDATED_DURATION_IN_DAY)
-            .name(UPDATED_NAME)
-            .description(UPDATED_DESCRIPTION);
-        return eventActivity;
+
+    public static Event createEventEntity(EntityManager em) {
+        Event event = new Event();
+        event.setName(DEFAULT_EVENT_NAME);
+        event.setDescription(DEFAULT_EVENT_DESCRIPTION);
+        event.setRemarks(DEFAULT_EVENT_REMARKS);
+        event.setVenue(DEFAULT_EVENT_VENUE);
+        event.setStartDate(DEFAULT_EVENT_START_DATE);
+        event.setEndDate(DEFAULT_EVENT_END_DATE);
+        event.setFee(DEFAULT_EVENT_FEE);
+        event.setRequiredTransport(DEFAULT_EVENT_REQUIRED_TRANSPORT);
+        event.setStatus(DEFAULT_EVENT_STATUS);
+        return event;
+    }
+
+    public static EventActivityDTO createDefaultEventActivityDTO() {
+        EventActivityDTO eventActivityDTO = new EventActivityDTO();
+        eventActivityDTO.setDescription(DEFAULT_DESCRIPTION);
+        eventActivityDTO.setDurationInDay(DEFAULT_DURATION_IN_DAY);
+        eventActivityDTO.setName(DEFAULT_NAME);
+        eventActivityDTO.setStartDate(DEFAULT_START_DATE);
+        return eventActivityDTO;
+    }
+
+    public static EventActivityDTO createUpdateEventActivityDTO() {
+        EventActivityDTO eventActivityDTO = new EventActivityDTO();
+        eventActivityDTO.setEventId(Long.MAX_VALUE);
+        eventActivityDTO.setDescription(UPDATED_DESCRIPTION);
+        eventActivityDTO.setDurationInDay(UPDATED_DURATION_IN_DAY);
+        eventActivityDTO.setName(UPDATED_NAME);
+        eventActivityDTO.setStartDate(UPDATED_START_DATE);
+        return eventActivityDTO;
     }
```

Comment:
@kachun333 In the test case we have to involved all the entities that is being used in the relative Service Implementation class is it?